src/main.c

```diff
@@ -1152,11 +1153,15 @@ main(int argc, char *argv[])
     np2srv.server_dir = SERVER_DIR;
 
     /* process command line options */
-    while ((c = getopt(argc, argv, "dhVp:f:U::m:u:g:n:i:t:x:v:c:")) != -1) {
+    while ((c = getopt(argc, argv, "dhVp:f:FU::m:u:g:n:i:t:x:v:c:")) != -1) {
         switch (c) {
         case 'd':
             daemonize = 0;
             break;
+	case 'F':
+            daemonize = 0;
+            use_syslog = 1;
+            break;
```

Comment:
Note that the cases are ordered the same way as the options in `getopt`, please stick to that order.